src/graphql/types/object/business-account.ts

```diff
@@ -1,97 +1,129 @@
-import { CouldNotFindTransactionsForAccountError } from "@domain/errors"
-import { GT } from "@graphql/index"
-import { mapError } from "@graphql/error-map"
-import { connectionArgs, connectionFromArray } from "graphql-relay"
+import getUuidByString from "uuid-by-string"
```

Comment:
maybe we can delete this business account until we have clear support for it. it would limit the code duplication/overhead on working on such PR in the meantime?

Reply:
yep, you are right, this file is identical to consumer account and was not included in the schema https://github.com/GaloyMoney/galoy/pull/1465/commits/7a27835ff7d528473a4e67248f506f87f03228ec